### DISCUSSION TITLE
[Enhancement] Leverage android architecture components

### DISCUSSION THREAD

**Discussion by J-Swift** [dotnet/maui#191] [General]
## Summary
In my experience on the native dev side, using `LifecycleOwner` and `ViewModel` from android architecture components really reduces the number of android quirks you run into and could help align ios/android closer together so are fewer use-cases to worry about from a cross-platform perspective

**Comment by softlion** [comment]
```c#
var lifecycleOwner = (ILifecycleOwner)Android.App.Application.Context;
```
